src/ImageSharp/Processing/Processors/Convolution/BokehBlurProcessor{TPixel}.cs

```diff
@@ -127,7 +127,7 @@ protected override void OnFrameApply(ImageFrame<TPixel> source)
                     in verticalOperation);
 
                 // Compute the horizontal 1D convolutions and accumulate the partial results on the target buffer
-                var horizontalOperation = new ApplyHorizontalConvolutionRowOperation(sourceRectangle, processingBuffer, firstPassBuffer, kernel, parameters.Z, parameters.W);
+                var horizontalOperation = new BokehBlurProcessor.ApplyHorizontalConvolutionRowOperation(sourceRectangle, processingBuffer, firstPassBuffer, kernel, parameters.Z, parameters.W);
```

Comment:
Can you not reference the processor passed to the constructor?

Reply:
Not sure what you mean - this is a static reference, we're not accessing an instance member. That's just the qualified type that's declared within `BokehBlurProcess`, so we need the parent type name when using it 🤔

Reply:
I misread, sorry, carry on. 👍